lib/exception_notifier/modules/formatter.rb

```diff
@@ -0,0 +1,114 @@
+require 'active_support/core_ext/time'
+require 'action_dispatch'
+
+module ExceptionNotifier
+  class Formatter
+    include ExceptionNotifier::BacktraceCleaner
+
+    attr_reader :app_name
+
+    def initialize(exception, opts = {})
+      @exception = exception
+
+      @env = opts[:env]
+      @errors_count = opts[:accumulated_errors_count].to_i
+      @app_name = opts[:app_name] || rails_app_name
+    end
+
+    #
+    # :warning: Error occurred in production :warning:
+    # :warning: Error occurred :warning:
+    #
+    def title
+      env = Rails.env if defined?(::Rails) && ::Rails.respond_to?(:env)
+
+      if env
+        "⚠️ Error occurred in #{env} ⚠️"
+      else
+        '⚠️ Error occurred ⚠️'
+      end
+    end
+
+    #
+    # A *NoMethodError* occurred.
+    # 3 *NoMethodError* occurred.
+    # A *NoMethodError* occurred in *home#index*.
+    #
+    def subtitle
+      errors_text = errors_count > 1 ? errors_count : 'A'
```

Comment:
What about having errors_text differentiate `A/An`?

I'd do:

```rb
errors_text = if errors_count > 1
                errors_count
              else
                exception_class.to_s =~ /^[aeiou]/i ? 'An' : 'A'
              end
```

What do you think?

Reply:
good idea, changed here df864ea68405a49a213fa0d0ca839983af31d0cc
thanks!